packages/graph/hash_graph/lib/graph/src/shared/identifier/mod.rs

```diff
@@ -15,6 +15,12 @@ pub mod ontology;
 
 pub type Timestamp = DateTime<Utc>;
 
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq, Serialize, Deserialize, ToSchema)]
```

Comment:
Why put `ToSchema` on this for now? Appreciate it may not hurt to do so but I think we should figure out the OpenAPI spec for this stuff in one go when consciously doing it. The `EntityVersion` one is overridden so I presume we don't need to do this?

Reply:
Good point, I removed it for now until we adjust the backend: 335193c558819a6b09279c87a1834e6774588c5b